diff --git a/scripts/fetch-prebuilt-helper.js b/scripts/fetch-prebuilt-helper.js
index e92d6e6130e79f95753bf97f46028c21d25ab1af..7177bf0868f9a9a123150541889ec3241756a4e4 100644
--- a/scripts/fetch-prebuilt-helper.js
+++ b/scripts/fetch-prebuilt-helper.js
@@ -21,8 +21,8 @@ if (fs.existsSync(helperLocation)) {
 const found = whereis(helperName)
 if (found !== '') {
   testExecutable(found)
-  fs.createReadStream(found).pipe(fs.createWriteStream(helperLocation));
-  fs.chmodSync(helperLocation, '500')
+  fs.copyFileSync(found, helperLocation)
+  fs.chmodSync(helperLocation, '775')
   // we need to copy so it's bundled correctly by electron-builder
   // sorry - i also hate mixing assumptions like this...
   console.warn('systrayhelper already installed - copied', found)
@@ -101,9 +101,9 @@ function testExecutable(path) {
 }
 
 function cleanup(path) {
+  fs.copyFileSync(path, helperLocation)
+  fs.chmodSync(helperLocation, '775')
   try {
-    fs.createReadStream(path).pipe(fs.createWriteStream(helperLocation));
-    fs.chmodSync(helperLocation, '500')
     fs.unlinkSync(tmpDownload)
     fs.unlinkSync(tmpUnpack)
     console.log('cleanup down. the helper is here:', helperLocation)
diff --git a/util.js b/util.js
index 3de0e73b2ace30882be82ba8bad407e238da4049..1a3d8ee90a3f2233ad1f60ce5dbbfa0e00044f34 100644
--- a/util.js
+++ b/util.js
@@ -6,14 +6,6 @@ const { homedir } = require('os')
 github repo was removed somehow so no way to give feedback/PRs
 therefore copied it here */
 const whereis = (filename) => {
-  const pathSep = process.platform === 'win32' ? ';' : ':'
-  const directories = process.env.PATH.split(pathSep)
-  for (var i = 0; i < directories.length; i++) {
-    var path = directories[i] + '/' + filename
-    if (existsSync(path)) {
-      return path
-    }
-  }
   // impure check if we already fetched the helper
   if (existsSync(helperLocation)) {
     return helperLocation
